admin/base/settings/defaults.py

```diff
@@ -219,3 +224,10 @@
 
 ENTRY_POINTS = {'osf4m': 'osf4m', 'prereg_challenge_campaign': 'prereg',
                 'institution_campaign': 'institution'}
+
+
+# Change desk settings in local.py
+DESK_KEY = ''
+DESK_KEY_SECRET = ''
+DESK_TOKEN = ''
```

Comment:
+1 to Matt's suggestion. I wouldn't consider it a blocker to getting this PR merged. @Ghalko Could you create an improvements ticket for a followup PR implementing this?